pkg/log/messages/info_messages.go

```diff
@@ -13,10 +13,12 @@ package messages
 */
 
 const CSPFK001I string = "CSPFK001I Authenticating as user '%s'"
-const CSPFK002I string = "CSPFK002I Creating Conjur client..."
-const CSPFK003I string = "CSPFK003I Retrieving following secrets from Conjur: %v"
-const CSPFK004I string = "CSPFK004I Creating Kubernetes client..."
-const CSPFK005I string = "CSPFK005I Retrieving Kubernetes secret '%s' from namespace '%s'..."
+const CSPFK002I string = "CSPFK002I Creating DAP/Conjur client"
```

Comment:
is this something that PO requested?

Reply:
Yes we needed to find a way to support both flows. 
@InbalZilberman

Reply:
yuck